### PR TITLE
mysql-test, part 7

### PR DESCRIPTION
**Pull request by osheroff**
- fix engine=merge, union() parsing

@zendesk/rules

### CODE REVIEW

**Comment by jpeyper** [comment]
This is all very exciting!

how many parts are you expecting to have? and when will all these fixes come into a release?

**Comment by osheroff** [comment]
@jeyper, not sure.  Have fixed parse problems with 300 lines of SQL but still have 129 lines to go; draw your own conclusions.  A bunch of the errors are for sure that we don't support GIS extensions, and there's also a whole host of parsing PARTITION table stuff that I'm just not gonna deal with.

My plan is probably roughly to get this file mostly passing and then do a 1.0 release (maybe alongside the bootstrapping patches?) sometime early in the new year.  For the parsing issues where it looks more likely that someone is gonna run into it, I may cherry-pick those forward and do a patch release sooner.

File: src/test/java/com/zendesk/maxwell/schema/ddl/DDLParserTest.java [hunk 1]
```diff
@@ -464,17 +464,21 @@ public void generateTestFiles() throws Exception {
...
+				nFixed++;
 				fixed.write((sql + "\n").getBytes());
 			} catch ( Exception e) {
 				assertions.add(sql);
 				problems.write((sql + "\n").getBytes());
+				nErr++;
 				System.err.println(sql);
 			}
 		}
+		System.out.println(nFixed + " fixed, " + nErr + " remain.");
```
Comment (vanchi-zendesk) [inline src/test/java/com/zendesk/maxwell/schema/ddl/DDLParserTest.java]: Remember to remove.

**Comment by vanchi-zendesk** [comment]
:+1:

**Pull request merged**